app/tests/reader_studies_tests/test_views.py

```diff
@@ -958,3 +957,105 @@ def test_ground_truth_view(client):
     )
 
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+@override_settings(task_eager_propagates=True, task_always_eager=True)
+def test_ground_truth_from_answers_workflow(client):
```

Comment:
Sort of an integration test. If for instance, one form doesn't set the permissions correctly another form would fail: this checks if they integrate correctly.